master/buildbot/test/unit/data/test_resultspec.py

```diff
@@ -23,60 +27,111 @@
 from buildbot.data.resultspec import NoneComparator
 from buildbot.data.resultspec import ReverseComparator
 
+if TYPE_CHECKING:
+    from typing import ClassVar
+    from typing import Sequence
+
+
+class ResultSpecMKListMixin:
+    @staticmethod
+    def mkdata(fld: Sequence[str] | str, *values):
```

Comment:
Maybe just have free function? It doesn't use self, so mixin is possibly an overkill.

Reply:
Ah ok, it's used below for inheritance.